### PR TITLE
Eventtests

### PR DESCRIPTION
**Pull request by kulisfunk**
added 3 tests, two for closure of popup and one to prove hover div created hidden

### CODE REVIEW

File: client/src/models/tests/integration/tests.js [hunk 1]
```diff
@@ -9,17 +9,44 @@ describe('space oddity functionality', function() {
...
     expect(popup.getCssValue('display')).to.eventually.equal('block');
   });
+
+  it('should set display to none for planet popup on out of popup window click event', function(){
+    var popup = element(by.css('.popup'));
+    var planet = element(by.css('#circle_Earth'));
+    var pluto = element(by.css('#circle_Pluto'));
+    planet.click();
+    browser.actions().mouseMove({x: 400, y: 400}).doubleClick().perform();
+    // pluto.click();
```
Comment (JamieSK) [inline client/src/models/tests/integration/tests.js]: Would you mind deleting commented out stuff? Otherwise looks great, cheers

**Review by JamieSK — CHANGES_REQUESTED** [review]
Tiny change will keep things tidy IMO

File: client/src/models/tests/integration/tests.js (hunk 1 above)
Comment (kulisfunk) [reply client/src/models/tests/integration/tests.js]: yeah no problem

Comment (kulisfunk) [reply client/src/models/tests/integration/tests.js]: ok code tidied up

**Review by JamieSK — APPROVED** [review]

**Comment by JamieSK** [comment]
Thanks Martin

**Review by Findsay — APPROVED** [review]

**Pull request merged**